trento/xml/article_sap_trento.xml

```diff
@@ -962,12 +962,30 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
         </listitem>
         <listitem>
           <formalpara>
-            <title><guimenu>SAP instances</guimenu>:</title>
-            <para>it summarizes the status of the ABAP instances as returned by <command>sapcontrol</command>.</para>
+            <title><guimenu>Application instances</guimenu>:</title>
+            <para>it summarizes the status of the application instances as returned by <command>sapcontrol</command>.</para>
+          </formalpara>
+        </listitem>
+      </itemizedlist>
+      <para>
+        Complementing the Operating System layer, you also have information about the health status of the HA components, whenever they exit:
```

Comment:
Is "Operating System Layer" a specific SAP term? If not, it should be lower case:

```suggestion
        Complementing the operating system layer, you also have information about the health status of the HA components, whenever they exit:
```